vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/resources/META-INF/resources/frontend/comboBoxConnector.js

```diff
@@ -239,7 +239,10 @@ import { ComboBoxPlaceholder } from '@vaadin/combo-box/src/vaadin-combo-box-plac
                 comboBox.$server.confirmUpdate(id);
             });
 
-            customElements.whenDefined('vaadin-combo-box').then(tryCatchWrapper(() => {
+            comboBox.addEventListener('opened-changed', () => {
```

Comment:
Nit: to align with other listeners, should probably be wrapped in `tryCatchWrapper` as well.

Reply:
Done.